scripts/cibuild.d/00-build-jars.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e
+
+if [[ -n "${RF_DEBUG}" ]]; then
+    set -x
+fi
+
+function build_ingest_jar {
+    echo "Building ingest JAR"
+    docker-compose \
```

Comment:
For consistency, seems like we should have all of this on one line like the others, or change the others to be multi-line.